src/BackupChain.cs

```diff
@@ -19,26 +20,33 @@ public class BackupChain : IBackupChain
   {
     private readonly IList<BackupMetadata> _orderedBackups;
 
+    /// <summary>
+    /// Chain needs to be 1 full backup, 1 diff and then the logs in the order they were taken
+    /// Sometimes backups can be striped (i.e. split into multiple files) - so we need to handle these cases too
+    /// https://www.sqlservercentral.com/articles/getting-a-list-of-the-striped-backup-files
+    /// </summary>

```

Comment:
You don't need to doc comment on every method or comment each line of code. 
My viewpoint on this is that if a function or code requires a doc comment to explain what it's doing, then the function is too complicated and needs to be broken up.

Reply:
This was more for a notice that this function can also handle striped backups in the chain
Will make it a single line comment and shorter just to state that point